shell/platform/common/text_input_model.h

```diff
@@ -177,6 +193,14 @@ class TextInputModel {
   bool composing() const { return composing_; }
 
  private:
+  enum MoveOrSelect { Move, Select };
```

Comment:
~~Two quick requests:~~
~~1. `enum class MoveOrSelect {`~~
~~2. Prefix the enum values with `k` so `kMove` and `kSelect`.~~

Actually, I think the code will be clearer if we get rid of this enum and the MoveOrSelect* methods and simply implement these directly inthe Move* and Select* methods. That avoids an extra level of indirection and the condition check.

Reply:
Thanks, done